packages/terra-search-field/src/SearchField.jsx

```diff
@@ -254,6 +254,7 @@ class SearchField extends React.Component {
             placeholder={placeholder}
             onChange={this.handleTextChange}
             disabled={isDisabled}
+            ariaLabel={inputText}
```

Comment:
If users set a custom aria-label via `<SearchField inputAttributes={{'aria-label': 'Custom aria label'}} />`, we'll want to pass that down to the `<Input />` component's `ariaLabel` prop.

We can update this line:
```
const inputText = intl.formatMessage({ id: 'Terra.searchField.search' });
```
to use the translation fallback if `aria-label` is not set via `inputAttributes` prop and then pass inputText as you have it on line 257.

Something like:
```
const inputText = inputAttributes.hasOwnProperty('aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' }); 
```